backend/pkg/config/config.go

```diff
@@ -233,6 +233,43 @@ func DefaultKubeConfigPersistenceFile() (string, error) {
 	return filepath.Join(kubeConfigDir, "config"), nil
 }
 
+// collectMultiConfigPaths looks at the default dynamic directory
+// (e.g. ~/.config/Headlamp/kubeconfigs) and returns any files found there.
```

Comment:
While it's useful to have the paths in the docs, this creates a possibility of the docs becoming wrong if we change the default kubeconfigs dir.